<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetMotion</class>
 <widget class="QDialog" name="SetMotion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>416</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Adobe 楷体 Std R</family>
    <pointsize>18</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>SetMotion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton{border-style: flat;}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>631</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 楷体 Std R</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>IO input</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>631</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QToolButton" name="toolButton_in1">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_in2">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/green.jpg</normaloff>:/CameraDtecte1/Resources/green.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_in3">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_in4">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_14">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_15">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_16">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_17">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>631</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 楷体 Std R</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>伺服电机</string>
   </property>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>电机一</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>电机二</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>速度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonplus">
       <property name="text">
        <string>正方向</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtondiv">
       <property name="text">
        <string>负方向</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>距离：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="icon">
     <iconset resource="CameraDtecte1.qrc">
      <normaloff>:/CameraDtecte1/Resources/公司LOG.jpg</normaloff>:/CameraDtecte1/Resources/公司LOG.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 楷体 Std R</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>IO output</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="toolButton_out1">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_out2">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/green.jpg</normaloff>:/CameraDtecte1/Resources/green.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_out3">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_out4">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_out5">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_9">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_7">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_5">
       <property name="styleSheet">
        <string notr="true">border-style: flat;</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="icon">
        <iconset resource="CameraDtecte1.qrc">
         <normaloff>:/CameraDtecte1/Resources/red.png</normaloff>:/CameraDtecte1/Resources/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CameraDtecte1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton_out1</sender>
   <signal>clicked()</signal>
   <receiver>SetMotion</receiver>
   <slot>changestate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_out2</sender>
   <signal>clicked()</signal>
   <receiver>SetMotion</receiver>
   <slot>changestate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_out3</sender>
   <signal>clicked()</signal>
   <receiver>SetMotion</receiver>
   <slot>changestate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_out4</sender>
   <signal>clicked()</signal>
   <receiver>SetMotion</receiver>
   <slot>changestate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>SetMotion</receiver>
   <slot>getmachinery(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>SetMotion</receiver>
   <slot>getmachinery(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>-9</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SetMotion</receiver>
   <slot>getspeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SetMotion</receiver>
   <slot>startmotion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SetMotion</receiver>
   <slot>savequit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changestate()</slot>
  <slot>getmachinery(bool)</slot>
  <slot>getspeed()</slot>
  <slot>startmotion()</slot>
  <slot>savequit()</slot>
 </slots>
</ui>
